api/api.proto

```diff
@@ -3763,6 +3781,39 @@ message SdkVolumeSnapshotScheduleUpdateRequest {
 message SdkVolumeSnapshotScheduleUpdateResponse {
 }
 
+// Defines the request to watch a volume
+message SdkWatchRequest {
+  // one of event type
```

Comment:
To expand on Luis's comments, maybe expand on some of these comments as it's autogenerated for API docs